--- a/ld/configure.tgt
+++ b/ld/configure.tgt
@@ -974,6 +974,10 @@
 			tdir_elf_i386=`echo ${targ_alias} \
 			    | sed -e 's/x86_64/i386/'`
 			;;
+x86_64-*-pranaos*)
+			targ_emul=elf_x86_64_pranaos
+			targ_extra_emuls=elf_i386_pranaos
+			;;
 x86_64-*-pe | x86_64-*-pep) targ_emul=i386pep ;
 			targ_extra_emuls=i386pe ;
 			targ_extra_ofiles="deffilep.o pep-dll.o pe-dll.o"
@@ -1071,6 +1075,10 @@
   NATIVE_LIB_DIRS='/usr/local/lib /usr/ccs/lib /lib /usr/lib'
   ;;
 
+*-*-pranaos*)
+  NATIVE_LIB_DIRS='/usr/lib/include'
+  ;;
+
 esac
 
 case "${target}" in
